bridges/otelzap/core.go

```diff
@@ -123,14 +125,48 @@ func (o *Core) Sync() error {
 	return nil
 }
 
-// TODO
 // Check determines whether the supplied Entry should be logged using core.Enabled method.
 func (o *Core) Check(ent zapcore.Entry, ce *zapcore.CheckedEntry) *zapcore.CheckedEntry {
+	if o.Enabled(ent.Level) {
+		return ce.AddCore(ent, o)
+	}
 	return ce
 }
 
 // TODO
 // Write method encodes zap fields to OTel logs and emits them.
 func (o *Core) Write(ent zapcore.Entry, fields []zapcore.Field) error {
+	r := log.Record{}
+	r.SetTimestamp(ent.Time)
+	r.SetBody(log.StringValue(ent.Message))
+	r.SetSeverity(getOTelLevel(ent.Level))
+	// TODO: Handle attributes passed via fields (exceptions: context.Context and zap.Namespace).
+	// TODO: Handle attributes passed via With (exceptions: context.Context and zap.Namespace).
+	// TODO: Handle context.Context containing trace context.
+	// TODO: Handle zap.Namespace.
+	// TODO: Handle logger name.
+	o.logger.Emit(context.Background(), r)
 	return nil
 }
+
+// converts zap level to OTel log level.
+func getOTelLevel(level zapcore.Level) log.Severity {
```

Comment:
1. no need to have documentation for such trivial unexplored function
2. let's name the function better

```suggestion
func convertLevel(level zapcore.Level) log.Severity {
```